src/modules/ContentList/index.tsx

```diff
@@ -14,22 +14,27 @@ import { useInterval } from '@aiera/client-sdk/lib/hooks/useInterval';
 import { Message, useMessageListener } from '@aiera/client-sdk/lib/msg';
 import { Content } from '@aiera/client-sdk/modules/Content';
 import { ChangeHandler } from '@aiera/client-sdk/types';
-import { ContentListQuery, ContentListQueryVariables, ContentType } from '@aiera/client-sdk/types/generated';
+import {
+    Content as ContentModel,
```

Comment:
I don't think you actually want to use `Content` or `Search` types here directly. Those types define all the fields available on those objects, the results you are using depend on what was actually queried in the GQL request. Using this will make it look like certain fields are available on the object for autocomplete/type checking when they actually aren't (and is the reason you have to cast below with `... as Search`). Instead, I think you want to create a new type alias from the `ContentListQuery` type. You can see an example of how we do this for events here: https://github.com/aiera-inc/aiera-client-sdk/blob/new-search/src/modules/EventList/index.tsx#L43